certified-connectors/AIForged/apiProperties.json

```diff
@@ -0,0 +1,66 @@
+{
+  "publisher": "Larc AI (PTY) Ltd",
+  "stackOwner": "Larc AI (PTY) Ltd",
+  "contact": {
+    "name": "AIForged",
+    "url": "https://www.aiforged.com",
+    "email": "info@aiforged.com"
+  },
+  "iconBrandColor": "#0E76BC",
+  "capabilities": [
+    "actions"
+  ],
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "AIForged",
+          "clientId": "swaggerui",
+          "clientSecret": "read the docs",
+          "scopes": [
+            "openid",
+            "email",
+            "phone",
+            "profile",
+            "roles",
+            "aiforgedapi"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          }
+        },
+        "uiDefinition": {
+          "displayName": "Log in with AIForged Account",
+          "description": "Log in with AIForged Account",
+          "tooltip": "Log in with AIForged Account",
+          "constraints": {
+            "required": "true"
+          }
+        },
+        "customParameters": {
+          "discoveryUrl": {
+            "value": "https://portal.aiforged.com/.well-known/openid-configuration"
+          },
+          "authorizationUrl": {
+            "value": "https://portal.aiforged.com/connect/authorize"
+          },
+          "tokenUrl": {
+            "value": "https://portal.aiforged.com/connect/token"
+          }
+        },
+        "identityProvider": "oauth2",
+        "properties": {
+          "IsFirstParty": "False"
+        },
+        "redirectMode": "Global",
+        "redirectUrl": "https://global.consent.azure-apim.net/redirect",
```

Comment:
@aiforged Can you please let us know the reason to add "redirectMode" and "redirectUrl" properties twice in the apiProperties.json? 

Can you please let us know if you have already tested the connector in your environment? If yes, please share a few screenshots for successful connection and flow runs.

Reply:
ok, I'll remove the duplicate

See the screenshots of the custom connector in PowerAutomate
<img width="277" alt="image" src="https://user-images.githubusercontent.com/25734528/149193957-fa6725c0-2a43-4ab4-b816-23ed8adad2ee.png">

Connector Authenticating
<img width="950" alt="image" src="https://user-images.githubusercontent.com/25734528/149194476-3c757b20-4dd3-43af-8b98-ba8b8e8f0aee.png">

Successfull execution of flow using connector
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/25734528/149194702-81ba8f30-f6ff-4ee5-a5c9-6f8329fbae6d.png">